### DISCUSSION TITLE
All my osu!laser data has been deleted

### DISCUSSION THREAD

**Discussion by HoldMyMind** [ppy/osu#23694] [Q&A] [answered]
When i was editing my osu layout skins my game crashed and after i run it again All my data (~20 skins and 5000 beatmaps) was deleted... When I check the files folder they take up 12 GB, I guess my data is still there, but it doesn't show up in the game. i really Need help. Someone PLEASE HELP ME!

My logs: https://www.mediafire.com/file/4q1uw89l2onn9nc/logs.rar/file

p/s: I tried running the game again but it didn't work (When I press f5, the osu lazer skips to the next track and doesn't refresh, i have no idea how to refresh beatmaps in osu!lazer)

**Comment by peppy** [comment]
There's no such thing as "refreshing beatmaps".

Please upload any `client.realm` files from your osu! data folder, and if possible also provide a screenshot of the folder showing what files are currently in it.

**Reply by HoldMyMind** [reply]
> There's no such thing as "refreshing beatmaps".
> 
> Please upload any `client.realm` files from your osu! data folder, and if possible also provide a screenshot of the folder showing what files are currently in it.

![image](https://github.com/ppy/osu/assets/134986976/dbf067e7-6faa-48d9-a4aa-2b840a803676)

cilent.realm https://www.mediafire.com/file/lh90ly815r8061h/client.realm/file

(My english is bad, sorry)

**Reply by peppy** [reply, marked as answer]
Could you please also share `client_1685359037_corupt.realm`? And after you've shared it (to make sure you have a backup copy), can you try deleting `client.realm` and renaming `client_1685359037_corupt.realm` to `client.realm`?

**Reply by HoldMyMind** [reply]
> Could you please also share `client_1685359037_corupt.realm`? And after you've shared it (to make sure you have a backup copy), can

HOLY SHIT YOU SAVE MY LIFE, IT WORKED, THANK YOU SO MUCH!

**Reply by peppy** [reply]
Great to hear. Unfortunately the log files you provided were after multiple restarts so it's hard to understand how this happened in the first place.

**Comment by Halozhan** [comment]
![image](https://github.com/ppy/osu/assets/25768818/1c136a56-0565-47f7-99aa-15d6f1309072)
![image](https://github.com/ppy/osu/assets/25768818/503d57ae-4c11-493e-bcf8-ba339198719b)
@peppy 
my client data is also corrupted too.
in my case, replacing a corrupted one that takes up 63MB to client.realm does not work for me.

**Reply by peppy** [reply]
Please send the file to pe@ppy.sh.
